data/alpha/register/social-housing-provider.yaml

```diff
@@ -1,6 +1,9 @@
 fields:
 - social-housing-provider
 - name
+- organisation
```

Comment:
if the social housing provider appears in another register e.g. as a local authority we don't want to duplicate data. therefore for those instances we curie the register containing that information from within the organization field, it contains entries such as local-authority-eng:BIR we can't do these curies in the name field as you can't have both curies and strings in the same field. 

cant put them both in the same field so we seperated into two, the second being organisation